README.md

```diff
@@ -0,0 +1,61 @@
+# Destiny Scrims Tracker #
+Version 1.0.0
```

Comment:
No way we're at `1.0.0` yet hombre! More like `0.0.1`

Reply:
waaat

Reply:
1.0.0 implies a solid interface and functionality. 0.0.1 implies things are changing rapidly, don't rely on anything.

Reply:
😂 na i know just forgot to change it